code/modules/power/singularity/narsie.dm

```diff
@@ -61,7 +61,7 @@
 	return
 
 /obj/singularity/narsie/proc/godsmack(atom/A)
-	if(isobj(A))
+	if(istype(A,/obj/))
 		var/obj/O = A
```

Comment:
Ding ding ding @duncathan gets it.

I'll change this back if doesn't screw with oldcult.